packages/chrome/package.json

```diff
@@ -24,8 +24,13 @@
     "@zendeskgarden/container-accordion": "^1.0.3",
     "@zendeskgarden/container-keyboardfocus": "^0.4.7",
```

Comment:
The `accordion` is being used by `CollapsibleSubNavItem`, so we only removed `keyboardfocus` since it's not being used anywhere.

These containers were not additional to `Sheet` and were already here.